qiskit/pulse/instruction_schedule_map.py

```diff
@@ -158,6 +159,8 @@ def get(self,
         schedule_generator = self._map[instruction].get(_to_tuple(qubits))
 
         if callable(schedule_generator):
+            params = [float(v) for v in params]
+            kwparams = {k: float(v) for k, v in kwparams.items()}
```

Comment:
Technically, this line isn't covered by a unit test (just the `params`, but not `kwparams`), so I'm happy to add one if you think it's worth it.